package/main/daml/ContingentClaims.Core.V3/daml.yaml

```diff
@@ -1,15 +1,15 @@
 # Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-sdk-version: 2.10.0
+sdk-version: 3.4.8
 name: contingent-claims-core-v3
 source: daml
-version: 3.0.0
+version: 3.0.1
```

Comment:
We'll need to bump the major version for all packages. Ie, in this case we'd need to set the version as `4.0.0`. 

Given the comment below, we should to set this something like you've done for `daml-ctl` - `3.99.0.20251127.0`